tests/framework/integration/testing.go

```diff
@@ -84,6 +99,21 @@ func BeforeTest(t testutil.TB, opts ...TestOption) {
 		testutil.RegisterLeakDetection(t)
 	}
 
+	if options.failpoint != nil && len(options.failpoint.name) != 0 {
+		err := gofail.Enable(options.failpoint.name, options.failpoint.payload)
+		switch {
+		case err == gofail.ErrNoExist:
+			t.Skip(fmt.Sprintf("skip test %s due to error (%v); please run 'make gofail-enable' before running the test", t.Name(), err))
```

Comment:
```suggestion
			t.Skipf("skip test %s due to error (%v); please run 'make gofail-enable' before running the test", t.Name(), err)
```

Reply:
There is no `t.Skipf`. I think it's overkill to add t.Name() given each test name will show up in the test output/report regardless. Updated in the latest rev.